source/index.ts

```diff
@@ -2,7 +2,7 @@
 /// <reference lib="dom"/>
 /// <reference types="node"/>
 
-export type Class<T = unknown> = new (...args: any[]) => T;
+import {Class, TypedArray, ObservableLike, Primitive} from './types';
```

Comment:
You need to export these again as they were previously exported to end-users.

Reply:
@sindresorhus Fixed!